Testing Vite revert

Nikhil-Ashoka opened the pull request:
- Reverted Vite config and env based config

Nikhil-Ashoka committed:
Testing Vite revert

- Reverted Vite config and env based config

Signed-off-by: Nikhil Ashoka <a.nikhil@ibm.com>

tiwari-nishant commented on vite.config.js:
```diff
@@ -1,48 +1,57 @@
-const isDev = process.env.NODE_ENV === 'development';
 const CWD = process.cwd();
-const DEV_ENV_CONFIG = loadEnv('development', CWD);
+const DEV_ENV_CONFIG = loadEnv('developement', CWD);
```
Development is misspelled

Nikhil-Ashoka replied:
Done

Nikhil-Ashoka committed:
Fixed infinite loop issue

- The UI was stuck when route between components.
- Updated the NavigationData.js

Nikhil-Ashoka committed:
Test by updating only Vite

- Test by updating only Vite and removing env

Nikhil-Ashoka committed:
Updated Vite and env config

- Fixed Vite and env based config
